### ISSUE TITLE
Search enhancements needed

### ISSUE THREAD

**Issue by GemmaNash** [ushahidi/Ushahidi_Android#107]
On the latest version of the ushahidi app, our deployment is very hard to find. It will only come up if I write https://britishgeologicalsurvey.crowdmap.com into the search maps by URL bit. Have I missed a trick with some meta data somewhere as typing in british geological survey returns no results?
Please advise. Thanks. Gemma

**Comment by eyedol** [comment]
@GemmaNash This feature is for filtering a long map list. You might want to use the "Find Maps Around Me" feature. Which uses your current location to fetch maps around that location.

**Comment by GemmaNash** [comment]
I've just tried that now (and had tried it before too). My map doesn't appear. In fact even at 250km I don't get any maps appearing.

**Comment by GemmaNash** [comment]
Can you please re-open this issue?

**Comment by eyedol** [comment]
Make sure your phone get a GPS fix. I can see maps around me within 250km. Which android version are you running?

Also if you know your latitude and longitude values, what do you get when you replace your values with the ones in  the URL http://tracker.ushahidi.com/list/?lat=55.3&lon=-3.4&distance=250&units=km

![Screenshot_2013-03-25-18-20-40](https://f.cloud.github.com/assets/73175/299371/9f1b1d1c-9579-11e2-8788-e9ebe545ad18.png)

**Comment by GemmaNash** [comment]
Ok, so Ushahidi did an update yesterday and I am receiving maps now, but only if I put my GPS on before I open the app. However, it's still not picking up my map and when I tried to enter and view it by adding the URL manually, it crashed. My lat/long on my deployment was a bit out and I've updated it, but I still can't see it.
I'm on Android version 4.0.4.
http://tracker.ushahidi.com/list/?lat=52.878997&lon= -1.078088&distance=250&units=km I don't see my deployment. It is definitely listed publically.
